docs/src/main/asciidoc/security-oidc-auth0-tutorial.adoc

```diff
@@ -145,8 +145,8 @@ This is the only time during this tutorial when you are expected to manually sta
 The configuration and code update steps in the remaining sections of this tutorial are automatically observed and processed by Quarkus without you needing to restart the application manually.
 ====
 
-Open the browser and access the following URL:
-`http://localhost:8080/hello`
+Open the browser and access \http://localhost:8080/hello.
```

Comment:
Why the `\`? Don't you want it to be clickable?

Reply:
Michelle was proposing it in her review so I thought I'd make it consistent with the earlier changes made to the doc (other localhost URLs also start with `\`). I'd actually not mind, now that I think of it, make it clickable as it would be easier to test for users, thanks for pointing it out.
Let me tweak it now

Reply:
Other docs have clickable URLs too

Reply:
@gsmet @gastaldi I've made all the actionable links clickable, those one which Michelle proposed to make non clickable remain so - at that moment in the tutorial the users have not started the application in dev mode yet, it is only explained at what address the endpoint will be listening at.

Reply:
Sorry for the noise, Lint showed one other typo. I don't intend to update anything else right now